admin/partials/nginx-helper-general-options.php

```diff
@@ -206,7 +272,10 @@
 									<input type="checkbox" value="1" id="purge_homepage_on_edit" name="purge_homepage_on_edit" <?php checked( $nginx_helper_settings['purge_homepage_on_edit'], 1 ); ?> />
 									&nbsp;
 									<?php
-										_e( 'when a <strong>post</strong> (or page/custom post) is <strong>modified</strong> or <strong>added</strong>.', 'nginx-helper' );
+										printf(
+											wp_kses( '%1$s<strong>%2$s</strong>%3$s<strong>%4$s</strong>%5$s<strong>%6$s</strong>.', array( 'strong' => array(), ) ),
```

Comment:
Same for this as above. Escape final o/p.

Check for other places and update it.